content/shmem_symmetric_available.tex

```diff
@@ -0,0 +1,33 @@
+\apisummary{
+Returns the memory available on the symmetric heap in bytes.}
+
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_symmetric_available(size_t *available_size);
```

Comment:
Does this interface use a "return"-by-pointer-argument style because of potential future error-handling return codes? It's just my preference, but I'd rather do that as little as possible in the API. (I just find it ugly.) Since we are returning a `size_t` (that is, `unsigned long` on most platforms), we could use `SIZE_MAX` as the indicator for error.

Also, `shmem_symmetric_available` sounds awkward to me. ("Symmetric" is an adjective; I want a noun that indicates what exactly is available.) What about `shmem_symmetric_heap_available`?